activerecord/lib/active_record/associations.rb

```diff
@@ -504,7 +504,7 @@ def association_instance_set(name, association)
     #
     # == Customizing the query
     #
-    # \Associations are built from <tt>Relation</tt>s, and you can use the Relation syntax
+    # \Associations are built from <tt>Relation</tt>, and you can use the Relation syntax
```

Comment:
Actually, I mean't to incorporate @robin850 's suggestion to make if more simpler. 
```
Associations are built from <tt>Relation</tt> objects, ..

```